financial-dashboard-app/src/app/spending-overview/spending-overview.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="spending-overview-container">
```

Comment:
containers zou ik persoonlijk zetten in een "section" tag. Is meer semantisch. Div's zou ik alleen gebruiken voor styling redenen. Zoals de div met de class spendings.